py/desispec/workflow/desi_proc_funcs.py

```diff
@@ -82,10 +82,10 @@ def get_shared_desi_proc_parser():
     parser.add_argument("--starttime", type=str, help='start time; use "--starttime `date +%%s`"')
     parser.add_argument("--timingfile", type=str, help='save runtime info to this json file; augment if pre-existing')
     parser.add_argument("--no-xtalk", action="store_true", help='disable fiber crosstalk correction')
-    parser.add_argument("--system-name", type=str, default=batch.default_system(), help='Batch system name (cori-haswell, perlmutter-gpu, ...)')
+    parser.add_argument("--system-name", type=str, help='Batch system name (cori-haswell, perlmutter-gpu, ...)')
```

Comment:
I'm happy to remove the definition of the system name here, but should we explicitly set `default=None`?

Reply:
default=None is the default for any argparse argument regardless of type, so we don't have to explicitly specify that.  There are multiple other places where we also rely on the argparse default being None without explicitly specifying it, and in general I prefer to not specify a parameter unless we are actually changing what it would otherwise be, or concerned that that default might change in the future outside of our control.